metis-authentication/metis-authentication-rest/README.MD

```diff
@@ -0,0 +1,98 @@
+## Spring boot
+
+The application supports both environment variables and spring boot .properties files.
+
+### Spring configuration .properties
+
+When running the spring boot application we should, ideally, not create and use an
+`application.properties` inside the project so that the project can be easily re-cloned
+and to also avoid accidental submission of configuration into version control.
+
+This can be applied with an environment variable pointing to the appropriate .properties file
+`SPRING_CONFIG_LOCATION=file:///path/to/file.properties`
+It can be applied in Intellij from the:
+> Edit configurations -> Modify Options -> Environment Variables -> Add the SPRING_CONFIG_LOCATION variable
+
+### Environment variables
+
+Alternatively an environment variables file can be created.  
+Spring boot accepts environment variables that can be created using the same names as in the `.properties` file,
+with their names upper-cased and punctuation characters replaced with "_".
```

Comment:
A small visualization example would be nice

Reply:
I try to avoid adding binaries such as images to git.
If we want to do this, it's probably better to use git-lfs.
I can have a look at it today.

Reply:
Or do you mean an example of how an environment variable would look like in this case?

Reply:
How an environment variable would look like

Reply:
I see, thanks for the clarification! I've added an example.